obspy/core/tests/test_inventory.py

```diff
@@ -583,10 +581,10 @@ def test_read_invalid_filename(self):
         # try read_inventory() with invalid filename for all registered read
         # plugins and also for filetype autodiscovery
         formats = [None] + list(formats)
+        expected_error_message = exception_msg.format(doesnt_exist)
         for format in formats[:1]:
-            with pytest.raises(IOError) as e:
+            with pytest.raises(IOError, match=expected_error_message):
```

Comment:
This change introduced the new failure

Either try  `with pytest.raises(IOError, match=re.escape(expected_error_message)):` or revert the change or just check for `IOError` or `FileNotFoundError` without checking the text explicitely.

Reply:
No, 'pytest.importorskip' was skipping all the tests in this file before, I don't think the test ever passed. I will try to look at later today.